spec/decorator_spec.rb

```diff
@@ -0,0 +1,48 @@
+require_relative '../trimmer_decorator'
+require_relative '../capitalize_decorator'
```

Comment:
- [ Required ] Please consider separating the tests for these two classes `TrimmerDecorator` and `CapitalizeDecorator` into different files. It's not considered good practice when unit tests from more than one class are written on the same file. Kindly fix this.